ansible/roles/laptop_images/tasks/firewall_rules.yaml

```diff
@@ -1,16 +1,4 @@
 ---
-- name: Allow port 69 (nice) for TFTP
-  community.general.ufw:
-    rule: allow
-    proto: udp
-    port: "69"
-
-- name: Allow port 67 for DHCP
```

Comment:
should the DHCP rule stay?

Reply:
nope that's cleanup from when I changed the laptop images role to share the dhcp role.  forgot to remove that rule then.